### ISSUE TITLE
Make it composer installable

### ISSUE THREAD

**Issue by markushausammann** [krewenki/php-docraptor#10]
Even the most simple dependency deserves to be composer installable!

**Comment by krewenki** [comment]
Feel free to make it composer installable.  I've never used composer and don't much work in PHP anymore.

**Comment by janxious** [comment]
@krewenki Would you like to transfer ownership of this to the DocRaptor owners for maintenance? Otherwise we will probably fork it and deal with this stuff there.

**Comment by markushausammann** [comment]
I was just planning on "forking", resp. actually just starting a new project. What ETA do you have @janxious ?

**Comment by janxious** [comment]
@markushausammann It would probably take me multiple days to get it done since I haven't used composer either. If you're willing to make a PR against a forked project, I am happy to review and merge such a thing at your earliest convenience.

**Comment by markushausammann** [comment]
That's a good solution, I will make a PR against your fork, just tell me when it's ready. Using packagist, everything can be setup in a way that every time you tag a release, packagist automatically publishes the new package version. You might want to open a packagist.org account or I'll register it through mine. Packagist is the default package repository for composer, which is what gem is to ruby, just better ;).

**Comment by markushausammann** [comment]
Actually you should open your own packagist.org account because you need an API key to set the hook up on github.

**Comment by janxious** [comment]
Repo is gonna be here: https://github.com/expectedbehavior/php-docraptor

**Comment by markushausammann** [comment]
Would you like to make changes before I make the composer PR?

**Comment by janxious** [comment]
@markushausammann Yeah, I am pulling in some PRs from this repo and making a few changes. Almost there.

**Comment by janxious** [comment]
@markushausammann I have made all the changes to the EB version of this thing, so PR away!

**Comment by markushausammann** [comment]
@janxious can we chat somewhere, I think I'm ready with the PR and I've got a few more PRs planned, if you're interested.

**Comment by markushausammann** [comment]
And could you open the issue tracker?

**Comment by janxious** [comment]
Issue tracker is opened. Let's continue this chat over there.
